### ISSUE TITLE
Deprecate repost

### ISSUE THREAD

**Issue by paulyoung** [design-systems/regimen#6]
Apparently it was decided that we don't want to assert that a piece of content is a repost or not.

I think this makes sense.

**Comment by bergie** [comment]
What _might_ make sense is helper to assert whether an item is local (has `item.metadata.url`), external (only `item.metadata.isBasedOnUrl`) or non-linkable.

Something like:

```
expect(item).to.be.linkable
expect(item).to.be.local
expect(item).to.be.remote
```

**Comment by paulyoung** [comment]
@bergie: seems easily doable, not sure how it fits with recent changes.

Meaning that, I was under the impression that we probably want most sections to work in either of those cases as opposed to bailing out completely.

**Comment by paulyoung** [comment]
@bergie said:

> yeah, this is mostly to allow sections to specialize if they want. `linkable` would be the "either works" case

**Comment by bergie** [comment]
@paulyoung yes, most sections should not care whether content is local or remote and just use `linkable`. But having the `local`/`remote` helpers will allow sections to specialise if they so want
